transform/mattermost-analytics/models/staging/cws/_cws__models.yml

```diff
@@ -64,3 +64,23 @@ models:
       - name: is_terms_accepted
         description: Whether the user accepted the terms and conditions.
 
+  - name: stg_cws__marketing
+    description: |
+      CWS Marketing data.
+
+    columns:
+      - name: marketing_id
+        description: Unique Identifier for Marketing data.
+        tests:
+          - not_null
+      - name: email
+        description: The user's email
+        tags: ['pii']
+      - name: subscribed_content
```

Comment:
A test on expected values can be added here. This way if a new value is added, the build will fail and we'll be able to make any required changes.

Reply:
Added test here.